classes/genimage.bbclass

```diff
@@ -80,6 +80,7 @@ DEPENDS += "genimage-native"
 GENIMAGE_IMAGE_SUFFIX ?= "img"
 
 GENIMAGE_IMAGE_NAME = "${IMAGE_BASENAME}-${MACHINE}-${DATETIME}"
+GENIMAGE_IMAGE_FULLNAME = "${GENIMAGE_IMAGE_NAME}.${GENIMAGE_IMAGE_SUFFIX}"
```

Comment:
This line should not split `GENIMAGE_IMAGE_NAME` from its `[vardepsexclude]` setting. And `GENIMAGE_IMAGE_LINK_NAME` comes after it anyway, too.

Reply:
Fixed, and I've thrown in a GENIMAGE_IMAGE_LINK_FULLNAME as well as I ended up adding another use for that anyway.